src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -57,6 +57,11 @@
     private List<String> javafxLinkFlags = Arrays.asList("-Wl,--whole-archive",
             "-lprism_es2_monocle", "-lglass_monocle", "-ljavafx_font_freetype", "-Wl,--no-whole-archive",
             "-lGLESv2", "-lEGL", "-lfreetype");
+    private List<String> capFiles = Arrays.asList("AArch64LibCHelperDirectives.cap",
```

Comment:
Not a huge issue, but I'm not sure why we always keeping constants as lists. Those lists are immutable and do use more memory comparing to arrays.